src/IDBStore.js

```diff
@@ -38,24 +38,24 @@
         fileStore = db.createObjectStore(IDBStore.DB_STORE_NAME);
       }
     };
-    req.onsuccess = function() {
+    req.onsuccess = () => {
       db = /** @type {IDBDatabase} */ (req.result);
       // add to the cache
       IDBStore.dbs[name] = db;
       callback(null, db);
     };
-    req.onerror = /** @this{IDBOpenDBRequest} */ function(e) {
-      callback(this.error);
-      e.preventDefault();
+    req.onerror = function(event) {
```

Comment:
Why change it from `e`? Other places use that, so it might be better.

Reply:
I think `event` is more clear here since `e` could stand for error or event and we are specifically digging the `target.error` out of the event in this case.

Honestly, I couldn't get this fire in all my testing so I'm just going by documentation here.